### ISSUE TITLE
asdf-standard not initialized

### ISSUE THREAD

**Issue by stscieisenhamer** [spacetelescope/asdf#207]
When first cloning and doing a 
```
python setup.py install
```
`astropy-helpers` is initialized, but `asdf-standard` is not. I have to manually do
```
git submodule init
git submodule update
```
Is this expected? If not, not clear as to why both are not automatically initialized.

**Comment by embray** [comment]
This is expected, and normal.

There's no way (unforunately) that I know if in git to automatically initialize a submodule when cloning.  This is done in astropy-helpers as a special feature of the ah_bootstrap.py script.  (It has been asked if we could extend this script to also automatically initialize other submodules; I definitely wouldn't object to this but no one has provided a solution).

You can do this a little more concisely with `git submodule update --init`

**Comment by stscieisenhamer** [comment]
Thanks. Didn't know if there was some trick that had been implemented to get this to happen.

**Comment by embray** [comment]
It wouldn't be bad to put in the readme or something at least.

**Comment by nden** [comment]
Closed by #215
